assignments/callbacks.js

```diff
@@ -23,30 +23,51 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 
 function getLength(arr, cb) {
+  return(cb(arr.length));
```

Comment:
you could actually pass the entire array into the call back here, like this: 
function getLength(arr, cb) {
  console.log(cb(arr));
}